.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
[consider][minor] I think we could remove this preamble, it doesn't add much info other than to make clear this was auto generated.

Reply:
Improved [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/92/commits/f3abe1e1a49fd7c4e8cc8045d027e3e839e9522a).